Extensions/PlatformBehavior/tests/platformerobjectruntimebehavior.spec.js

```diff
@@ -2086,6 +2200,9 @@ describe('gdjs.PlatformerObjectRuntimeBehavior', function () {
     });
 
     it("can't walk from a platform to another one that is too high", function () {
```

Comment:
In the future, this may depend on the slope climbing factor, if I understood properly your idea from https://github.com/4ian/GDevelop/issues/3013?

(For now 1 pixel difference is the most to go from one platform to another, but in the future this could be dependent on how much we can go according to the slope climbing?)

Reply:
Yes, these 2 tests will have to change if the 1st solution is chosen.